server/Html.js

```diff
@@ -50,6 +72,7 @@ const Html = (props) => {
         <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
         {head.title.toComponent()}
         {head.meta.toComponent()}
+        <HotjarScript />
```

Comment:
Anbefalingen er å sette den til `<head>`, men den kan plasseres i `<body>` også, forutsatt at innholdet laster gjennomsnitlig raskt. Mer info her: https://docs.hotjar.com/docs/understanding-the-tracking-code.